### PR TITLE
make github/init

### PR DESCRIPTION
**Pull request by nitrocode**
## what
* make github/init

## why
* Bump context of the endpoint group

## references
N/A

### CODE REVIEW

File: README.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ license: "APACHE2"
 copyrights:
   - name: "Cloud Posse, LLC"
     url: "https://cloudposse.com"
-    year: "2021"
+    year: "2022"
```
Comment (aknysh) [inline README.yaml]: don't change the year here, it's the starting year

**Review by aknysh — COMMENTED** [review]
see comment

File: README.yaml (hunk 1 above)
Comment (nitrocode) [reply README.yaml]: That's what I get for going above and beyond lol... now I have to rerun the tests.

**Review by aknysh — APPROVED** [review]

**Pull request merged**